microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,253 @@
+%% *************************************************************************
+%%
+%% This is an RIT Space Exploration Standard defining guidelines for content
+%% and formatting of project design documents.
+%%
+%% This document uses IEEEtran.cls, the official IEEE LaTeX class
+%% for authors of the Institute of Electrical and Electronics Engineers
+%% (IEEE) Transactions journals and conferences.
+%%
+%% *************************************************************************
+
+%% *************************************************************************
+% LaTeX REFERENCES
+% ----------------
+%   Intro to LaTeX: http://www.rpi.edu/dept/arc/docs/latex/latex-intro.pdf
+%   Comprehensive LaTeX symbol list: http://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf
+%% *************************************************************************
+
+\documentclass[conference]{IEEEtran} % http://www.ctan.org/pkg/ieeetran
+\usepackage[euler]{textgreek}
+\usepackage{blindtext}
+\usepackage{graphicx}
+\usepackage{nomencl}
+\usepackage{siunitx}
+\usepackage{hyperref}
+\usepackage[T1]{fontenc}
+\usepackage{etoolbox}
+\makenomenclature{}
+
+\title{\textmu{}Hab}
+
```

Comment:
to save time you can write custom commands similar to `\LaTeX` that act as shortcuts or macros. One that might work for you is 
```latex
\newcommand{\uHab}{\textmu{}Hab}
```
Usage: `the name of my project is \uHab`
Output: `the name of my project is \textmu{}Hab`